sound/soc/sof/trace.c

```diff
@@ -252,8 +274,9 @@ EXPORT_SYMBOL(snd_sof_init_trace);
 int snd_sof_trace_update_pos(struct snd_sof_dev *sdev,
 			     struct sof_ipc_dma_trace_posn *posn)
 {
-	if (sdev->dtrace_is_enabled && sdev->host_offset != posn->host_offset) {
-		sdev->host_offset = posn->host_offset;
+	if (sdev->dtrace_is_enabled &&
+	    sdev->trace_offset != posn->host_offset) {
+		sdev->trace_offset = posn->host_offset;
 		wake_up(&sdev->trace_sleep);
```

Comment:
The code is becoming unreadable. Please keep the initial definition of host_offset and init_host_offset, the introduction of 'trace' makes things difficult to follow.

Reply:
I agree with Liam that 'trace' is better than 'host' here, as it is member of sdev, but only used for trace(not for other streams).

Reply:
then change the dma_trace_posn structure to use trace_offset... It makes no sense to use different fields in different structures when it's the same concept.

Reply:
the dma_trace_posn struct is replied from FW, looks host_offset better there. the sdev is only used in driver, so they have actually different standpoint, can we just keep this?

Reply:
I don't understand your reply. Why would the same concept be named differently if this is firmware or driver?